website/docs/components/form/select/partials/code/how-to-use.md

```diff
@@ -143,7 +155,7 @@ To show the user that their input is not valid, you have to do two things: decla
 </Hds::Form::Select::Field>
 ```
 
-##### Custom control ID
+#### Custom control ID
 
 In case it's necessary to have custom ID for the control, instead of the one automatically generated by the component (eg. because it needs to be referenced in the code for other reasons), you just need to pass a `@id` argument to the "field":
```

Comment:
```suggestion
In cases where it’s necessary to have custom id attribute for the control, instead of the one automatically generated by the component (e.g., because it needs to be referenced somewhere else in the code), you just need to pass a `@id` argument to the "field":
```